openverse_catalog/dags/data_refresh/record_reporting.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from common import slack
+
+
+log = logging.getLogger(__name__)
+
+
+def report_record_difference(before: str, after: str, media_type: str):
+    before = int(before)
+    after = int(after)
+    count_diff = after - before
+    percent_diff = (count_diff / before) * 100
+    # Note for formatting:
+    # '+' - number will always have a sign in front of it
+    # ',' - number is comma separated
+    # '.' - number is a float
+    message = f"""
+_Note: All values are row estimates and are not (but nearly) exact_
+*Record count difference for `{media_type}`*: {before:,} → {after:,}
+*Change*: {count_diff:+,} ({percent_diff:+}% Δ)
+"""
+    log.info(message)
+    if slack.should_send_message():
+        slack.send_message(text=message, username="Data refresh record difference")
```

Comment:
The slack utility already takes care of checking `should_send_message` [here](https://github.com/WordPress/openverse-catalog/blob/main/openverse_catalog/dags/common/slack.py#L225), so we shouldn't need to check.

I'm also updating `send_message` in #643 to automatically[ log the message](https://github.com/WordPress/openverse-catalog/pull/643/files#diff-4284b9bf552cdfb3dbbfa19c55fcb442f98d3933939dc34c25790834988c38edR225) (even if `should_send_message` fails), since I think we want that by default -- so we should be okay to remove the `log.info` here.

Reply:
Oh good! That's perfect, I'll remove all this then, thank you 😄